app/src/main/java/io/gnosis/safe/ui/base/activity/BaseActivity.kt

```diff
@@ -30,6 +30,9 @@ abstract class BaseActivity : AppCompatActivity() {
             tracker.logScreen(it)
         }
         settingsHandler.applyNightMode(settingsHandler.nightMode)
+        // in case app was updated in the background and restarted
+        settingsHandler.updateUpdateInfo()
+        settingsHandler.fetchRemoteConfig()
```

Comment:
Why both? Wouldn't `settingsHandler.fetchRemoteConfig()` be enough here? It calls `updateUpdateInfo()` when it is done fetching the remote config.

Reply:
if app was updated in the background current version would be different and flags in settings handler have to be updated. This covers the case when on app restart there is not internet connection or fetch fails. Otherwise app will be still regarded as deprecated, even though it was updated.